### DISCUSSION TITLE
Migrating from Raspbee II to slaesh’s CC2652RB

### DISCUSSION THREAD

**Discussion by m0wlheld** [Koenkk/zigbee2mqtt#5860] [General]
Assuming that [slaesh’s CC2652RB](https://www.zigbee2mqtt.io/information/supported_adapters.html#slaeshs-cc2652rb-stick-recommended-for-most-users) is going to be delivered today, I tend to migrate my coordinator hardware from Raspbee II add-on module to the new stick.

Do I need to do anything but plug-in slaesh’s CC2652RB (using extension cable of course) and change the `serial:` settings in configuration? Should I make some things in configuration explizit, what is currently on default (panID, extendedPanID)?

The need to re-pairing everything when changing coordinator hardware would be a nightmare. Want to avoid that for sure.

**Comment by m0wlheld** [comment]
according to the [FAQ](https://www.zigbee2mqtt.io/information/FAQ.html#how-do-i-migrate-from-a-cc2531-to-a-more-powerful-coordinator-eg-zzh), repairing is a must, especially as the CC2652RB uses Zigbee 3.0 firmware, but the Raspee II does not. 

Now, the problem (due to distance) is, that only 2 (routing) devices are connected to the coordinator, while the remaining 30+ are part of the mesh. Is there any chance of repairing without having to uninstall  every device?

How about this: 

* I install the CC2652RB additionally to the Raspberry, run a 2nd Z2M instance for it using a different channel and Pan ID. I enable permit_join on that instance. 
* I request unpairing / factory reset via the first Z2M instance, starting with the most distant devices to the closest ones. permit_join is already disabled. 

As soon as the closest devices pair to the CC2652RB on the new instance, remaining devices should be able to join, too - isn‘t it?

**Reply by dircr** [reply]
Hey @m0wlheld, did you manage to migrate? How did you do it?

**Reply by m0wlheld** [reply]
No, I kept the Raspbee II, no time to re-pair 40+ devices

**Reply by lux73** [reply]
> No, I kept the Raspbee II, no time to re-pair 40+ devices

you wanna sale the CC2652RB?? ;-)  would be definitely interested

**Reply by m0wlheld** [reply]
I used it on another location, works great :)

**Comment by jimmyeao** [comment]
Im doing this at the weekend, moving from a CC2531 to a ZZH CC2652. My question is, will re-pairing keep the exisiting IDs in the database?
If all my Homeassistant entities are going to change as a result, that would be a bit of a kicker.
